spec/models/user_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  describe 'validations for User model' do
+    before(:each) do
+      @user = User.new(name: 'Tom', bio: 'Teacher from Mexico', post_counter: 0)
+    end
+
+    before { @user.save }
+
+    it 'if there is name' do
+      @user.name = nil
+      expect(@user).to_not be_valid
+    end
+
+    it 'PostCounter must be greater than or equal to zero' do
+      @user.post_counter = -1
+      expect(@user).to_not be_valid
+    end
+
+    it 'PostCounter must be greater than or equal to zero' do
+      @user.post_counter = 5
+      expect(@user).to be_valid
+    end
```

Comment:
- [ ] Good job so far 👏 From the project guidelines, It is suggested to write `validation` tests together with other tests. In order to understand it better, let me give you an example for this `user_spec` 👇
```ruby 
describe 'validations' do
  it { should validate_presence_of(:name) }
end
```
You can add tests for other validations that you added in the model 😃 👍